src/backend/downloadmanager/downloadqueue.ts

```diff
@@ -97,6 +100,15 @@ async function initQueue() {
   }
 
   queueState = 'idle'
+  if (!isPaused() && isIdle() && getAutoShutdown() && didReallyDownload) {
+    logInfo(
+      'Auto shutdown enabled. Shutting down in 3s...',
+      LogPrefix.DownloadManager
+    )
+    setTimeout(() => {
+      shutdown()
+    }, 3000)
+  }
```

Comment:
we probably should show a dialog in the frontend with an option to cancel the shutdown, because if you start downloading a game and forgot to remove the check and start doing something else this will shutdown your computer without you knowing

Reply:
Dialog added and shutdown timer extended to 10 seconds instead of 3